src/Scripting/Compiler/GraphNodeRegistry.hpp

```
@@ -0,0 +1,995 @@
+
+					retBuffer[0] = false;
+				}
+			});
+
+			addDefault({
+				ControlFlowLoop,
+				"Sequence", "Control::Sequence",
+				{ Graph::ExecInID },
+				{ "Then 0", "Then 1", "Then 2" },
+				{}, {},
+				[](AsTL::OpaqueExecCtx *, const GraphNodeDescriptor *self, AsTL::IDX nodeID, std::string_view triggeredPin, std::span<const AsTL::StackValue> args, AsTL::StackValue* retBuffer) -> void {
```

Clang (14.0.6)

[note] nesting level increased to 1

(clang)